packages/notte-sdk/src/notte_sdk/endpoints/agents.py

```diff
@@ -256,7 +258,7 @@ def close(self, agent_id: str) -> AgentResponse:
         self._last_agent_response = None
         return response
 
-    def status(self, agent_id: str) -> AgentStatusResponse:
+    def status(self, agent_id: str, **data: Unpack[AgentStatusRequestDict]) -> AgentStatusResponse:
```

Comment:
can can you `agent_id` in  `AgentStatusRequestDict` to have some consistency with the rest of the types ?